docs/en/restful_api.md

```diff
@@ -0,0 +1,112 @@
+# Restful API
+
+### Launch Service
+
+```shell
+python lmdeploy/serve/openai/api_server.py ./workspace server_name server_port
```

Comment:
Following this launching command, I got error like below:
```
```

Reply:
Fixed